src/modules/commander/commander_params.c

```diff
@@ -802,6 +802,21 @@ PARAM_DEFINE_INT32(NAV_RCL_ACT, 2);
  * Temporary Parameter to enable interface testing
  *
  * @boolean
+ * @reboot_required true
```

Comment:
why is the reboot required?

Reply:
Uh, that is actually a good point:
I was thinking in terms of having added OA at prearm checks, thus if you enable OA you are required to reboot to force the user to redo prearm checks.

But actually on other platforms (..) OA is a feature which can be enabled/disabled inflight. 
I think the answer to your question depends on how OA is meant to be in the future. @jkflying can maybe clarify

Reply:
@mrivi after a quick offline chat with @jkflying we decided to have the parameter as it was.
Thanks for raising it.
See here e004f77